py/torch_tensorrt/fx/fx2trt.py

```diff
@@ -257,6 +272,34 @@ def run(
         engine = self.builder.build_engine(self.network, builder_config)
         assert engine
 
+        import os
+        def get_file_name(org):
+            file_name = org
+            i = 0
+            while os.path.exists(os.path.abspath(file_name)):
+                i += 1
+                file_name = org + str(i)
+            return file_name
+
+        engine_file = os.environ.get('TORCH_FX_DUMP_ENGINE')
+        if engine_file:
+            dump_file = get_file_name(engine_file)
+            print(f'Dumping engine to {dump_file}')
+            s = engine.serialize()
+            with open(dump_file, 'wb') as f:
+                f.write(s)
+        engine_info_file = os.environ.get('TORCH_FX_DUMP_ENGINE_INFO')
```

Comment:
I believe that TRTModule / TRTModuleNext already support dumping engine info directly. I think this would be better outside of the compile pipeline. I could imagine a piece of code which takes in a compiled module and recursively finds all TRTModule / TRTModuleNext and hits the dump call.

Reply:
Got it.